sdk/storage/src/blob/blob/requests/source_content_md5.rs

```diff
@@ -1,12 +1,13 @@
-use crate::AddAsHeader;
+use azure_core::headers::SOURCE_CONTENT_MD5;
```

Comment:
I would move the constant off core as well since it's no longer used there (or maybe remove the constant altogether and just put the string in `add_as_header`). What do you think?

Reply:
That works for me.